.github/workflows/nodejs.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14, 16, 18]
+        node-version: [18]
```

Comment:
Should we add 20? I've been seeing more Actions migrate to it, and it's now a year old; if nothing breaks might be nice to start testing against it.

(disclaimer: I'm not the expert on node versions.)

Reply:
Ooh okay! let's see what it tells us!

Reply:
WOO! Added, and passed!